dev/devicelab/manifest.yaml

```diff
@@ -260,3 +260,19 @@ tasks:
       Measures the performance of Dart VM hot patching feature on Windows.
     stage: devicelab_win
     required_agent_capabilities: ["windows"]
+
+  # Production of sample catalog pages and screenshots
+
+  android_sample_catalog_generator:
```

Comment:
So far we've been grouping tasks by OS, all "has-android-device" are together, etc. Even though I like this better, let's keep it consistent. I'm ok with either keeping status quo, or regrouping the entire file, if you have extra cycles to do it.